app/jobs/delete_jobseekers_with_incorrect_emails.rb

```diff
@@ -0,0 +1,13 @@
+class DeleteJobseekersWithIncorrectEmails < ApplicationJob
+  queue_as :default
+
+  def perform
+    client = Notifications::Client.new(ENV.fetch("NOTIFY_KEY"))
+
+    failed_email_addresses = client.get_notifications(template_type: "email", status: "permanent-failure").collection.map(&:email_address)
+
+    failed_email_addresses.each do |failed_email|
+      Jobseeker.find_by(email: failed_email)&.destroy
+    end
```

Comment:
:raising_hand_man: 2 suggestions in this block:
1. If I read the code right it seems we could turn it into a single DB query instead of looping over the list (`where...destroy_all`)
2. As we are destroying the Jobseeker maybe we want to make sure that this only happens to unconfirmed users? Just to play safe :crossed_fingers:  (`confirmed_at: nil`).
```suggestion
    Jobseeker.where(email: failed_email_adresses, confirmed_at: nil).destroy_all
```